pkg/didcomm/common/service/messenger.go

```diff
@@ -28,12 +28,25 @@ type Messenger interface {
 	// ReplyTo replies to the message by given msgID.
 	// Keeps threadID in the *decorator.Thread.
 	// Using this function means that communication will be on the same thread.
-	ReplyTo(msgID string, msg DIDCommMsgMap)
+	ReplyTo(msgID string, msg DIDCommMsgMap) error
 
 	// Send sends the message by starting a new thread.
-	Send(msg DIDCommMsgMap, myDID, theirDID string)
+	Send(msg DIDCommMsgMap, myDID, theirDID string) error
```

Comment:
just curious: any option to send without connection(without myDID and theirDID), 
eg: `Destination` in outbound dispatcher interface?

Reply:
unfortunately, there is no such possibility. Not sure even how to implement it. I do not even know whether we have a destination from where this message came (e.g need for a reply). But I guess, we will come up with something.

Reply:
It is required for supporting the `~service` decorator, which is in AIP 1.0 (although there are [lingering concerns](https://github.com/hyperledger/aries-rfcs/issues/358#issuecomment-575873239)).